spring-cloud-gcp-autoconfigure/pom.xml

```diff
@@ -187,6 +187,10 @@
                     <groupId>io.grpc</groupId>
                     <artifactId>grpc-core</artifactId>
                 </exclusion>
+                <exclusion>
+                    <groupId>io.zipkin.zipkin2</groupId>
+                    <artifactId>zipkin</artifactId>
```

Comment:
Huh? Isn't that enough to just exclude on the `dependencyManagement` level in the root pom?

Reply:
I thought that didn't always work. It might be dependent on version of Maven. I'll give it a try.

Reply:
I think you're right! I dunno why we have the exclusions this way.